data-models/README.md

```diff
@@ -2,4 +2,43 @@
 
 Common Akenza data models that are used throughout the device types.
 
-- Also refer to: https://github.com/smart-data-models/CrossSector and https://webthings.io/api/
+Data models are referenced by using the id and relevant definition:
+
+```
+{
+  "$id": "https://akenza.io/decentlab/iam/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "temperature": {
+      "$ref": "https://akenza.github.io/device-types-library/data-models/ambiance/climate/schema.json#/$defs/temperatureC"
```

Comment:
```suggestion
      "$ref": "https://akenza.github.io/device-types-library/data-models/ambiance/climate/schema.json#/$defs/temperatureCelsius"
```

I agree to not nest it here (e.g. `.../$defs/temperature/celsius`), but I'd write out the scale to be as precise as possible (as not everyone is familiar with every scale that is not as obvious as celcius). 

What's your take on this?

Reply:
it makes sense to add a bit more verbosity, although this value is likely not being used by users